src/components/CardComponent/index.view.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import EachCard from '../EachCard';
+
+// importing styles like this ensures that
+// styles dont clash
+// prefixes them with a unique key
+// import * as styles from './index.module.css';
+
+const CardComponent = (props) => {
+	// Try to deconstruct your props
+	// It makes it easier to add default values
+	// because we wouldn't need to check if the
+	// value doesn't exist
+    const { mainArray } = props;
+	return (
+		<div>
+			{mainArray.map((eachCard, index) => (
+				<EachCard title = {eachCard.title} content = {eachCard.content}/>
```

Comment:
NIT for now: Pretty sure React warns us if `key` is not used with this mapping function. Its just a way to recognize components from each other. I think it prevents or helps prevent data leak and such.